includes/KSES.php

```diff
@@ -564,13 +565,49 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 			],
 		];
 
-		$allowed_tags = array_merge( $allowed_tags, $story_components );
+		$allowed_tags = $this->array_merge_recursive_distinct( $allowed_tags, $story_components );
 
 		$allowed_tags = array_map( [ $this, 'add_global_attributes' ], $allowed_tags );
 
 		return $allowed_tags;
 	}
 
+	/**
+	 * Recursively merge multiple arrays and ensure values are distinct.
+	 *
+	 * Based on information found in http://www.php.net/manual/en/function.array-merge-recursive.php
+	 *
+	 * @since 1.5.0
+	 *
+	 * @param array ...$arrays [optional] Variable list of arrays to recursively merge.
+	 *
+	 * @return array An array of values resulted from merging the arguments together.
+	 */
+	public function array_merge_recursive_distinct( array ...$arrays ) {
```

Comment:
Let's make this protected. No reason for it to be public.

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6542/commits/055bdbcac14e27cd8b382d8e5cd6c510262af304